app/controllers/lead_providers/partnership_csv_uploads_controller.rb

```diff
@@ -7,26 +7,37 @@ def new
     end
 
     def create
-      # TODO: do we want to catch empty session[:delivery_partner_id] ?
       if params[:partnership_csv_upload].blank?
         @partnership_csv_upload = PartnershipCsvUpload.new
         @partnership_csv_upload.errors[:base] << "Please select a CSV file to upload."
         render :new and return
       end
 
-      @partnership_csv_upload = PartnershipCsvUpload.new(upload_params.merge(
-                                                           lead_provider_id: current_user.lead_provider_profile.lead_provider.id,
-                                                           delivery_partner_id: session[:delivery_partner_id],
-                                                         ))
+      @partnership_csv_upload = PartnershipCsvUpload.new(
+        upload_params.merge(
+          lead_provider_id: current_user.lead_provider_profile.lead_provider.id,
+          delivery_partner_id: session[:delivery_partner_id],
+        ),
+      )
```

Comment:
I asked this on another PR and I still don't get it. Why does `upload_params` have parameters that we are always overriding here with `merge`?

Reply:
Yeah really not sure here. I'll see if I can simplify it

Reply:
I removed LP and DP from `upload_params`, since they're not in the form